src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -0,0 +1,29 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var ConsoleManagerRemote = require("text!lib/ConsoleManagerRemote.js");
+
+    function getRemoteScript() {
+        return "<script>\n" + ConsoleManagerRemote + "</script>\n";
+    }
+
+    function isConsoleRequest(msg) {
+        return msg.match(/^bramble-console/);
+    }
+
+    function handleConsoleRequest(args, type) {
+        // Add an indentifier to the front of the args list
+        args.unshift("[Bramble Console]:");
+       
+        // Time related arguments
+        if(type === "time" || type === "timeEnd") {
```

Comment:
I think you should handle this case in the remote code, where you're dealing with those methods vs. here, which is really far away from the code that originally caught it.  This code should be fairly agnostic about what to do with the message it's receiving, it should just handle it without modification.